src/Ocelot/Configuration/Validator/FileConfigurationFluentValidator.cs

```diff
@@ -107,8 +104,7 @@ private static bool AllRoutesForAggregateExist(FileAggregateRoute fileAggregateR
 
         private static bool IsPlaceholderNotDuplicatedIn(string upstreamPathTemplate)
         {
-            var regExPlaceholder = new Regex("{[^}]+}");
-            var matches = regExPlaceholder.Matches(upstreamPathTemplate);
+            var matches = _regExPlaceholder.Matches(upstreamPathTemplate);

```

Comment:
No catching and processing of the `RegexMatchTimeoutException` exception.

Will it result to unhandled exception by app domain?
Wil it be caught by upper context?